src/main.ts

```diff
@@ -58,6 +58,8 @@ Sentry.init({
   ],
   release: process.env.VERSION,
   tracesSampleRate: 1.0,
+  environment:
+    process.env.GITHUB_ACTION_REF === 'main' ? 'production' : 'development',
```

Comment:
is this an actual GHA env var? I don't see it here https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables

Reply:
I don't think it's documented:
https://github.com/getsentry/sentry/runs/6836956146?check_suite_focus=true#step:3:173

They were introduced in here:
https://github.com/actions/runner/pull/585